experimental/rules_python/wheelmaker.py

```diff
@@ -107,14 +107,15 @@ def arcname_from(name):
         # Find the hash and length
         hash = hashlib.sha256()
         size = 0
-        with open(real_filename, 'rb') as f:
-            while True:
-                block = f.read(2 ** 20)
-                if not block:
-                    break
-                hash.update(block)
-                size += len(block)
-        self._add_to_record(arcname, self._serialize_digest(hash), size)
+        if not os.path.isdir(real_filename):
```

Comment:
Would prefer an early `return`, before `hash` and `size` are assigned.

Reply:
updated